gsy_framework/schemas.py

```diff
@@ -32,7 +32,7 @@ class ScenarioSchemas:
                     "coefficient_percentage": {"type": ["number", "null"]},
                     "fixed_monthly_fee": {"type": ["number", "null"]},
                     "marketplace_monthly_fee": {"type": ["number", "null"]},
-                    "market_maker_rate": {"type": "number"},
+                    "market_maker_rate": {"type": ["number", "null"]},
```

Comment:
So I guess the market_maker_rate can be None, too right? Do you think it would be worth it to reflect this also in the typing [here](https://github.com/gridsingularity/gsy-e/blob/af681360fdb93b1793f9b8f799e636dc43b486f6/src/gsy_e/models/area/__init__.py#L264) ?

Reply:
I actually think that the way I changed the schema is not correct here. AFAIS, the "Utility Price" value (mapped to `market_maker_rate`) should always be provided in the community datasheet, so all datasheet items with `"type": "Area"` should have it. Not sure why I allowed `null` values at some point; probably it was because the sample datasheet that was provided in the ticket wasn't correct and had many missing values that should be required instead. I'll revert this change, thanks!